### PR TITLE
Fix/debug console errors

### LINKED ISSUE #93: BUG: Console Error - "Can't perform a React state update on an unmounted component"
- "Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function."
- StackOverflow article: https://stackoverflow.com/questions/54954385/react-useeffect-causing-cant-perform-a-react-state-update-on-an-unmounted-comp

### PR DESCRIPTION
**Pull request by pcmueller**
## Type of Change
- [ ] JSX
- [ ] HTML
- [x] JavaScript
- [x] CSS
- [x] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [x] Refactor/Fix Bugs

## Please describe the change made
Fixed the "memory leak / unmounted component" console error.

## Please give a short description of the changes that were made below

- Fix:
- Basically what this was telling us is that you can't update state on a component that's no longer being rendered - the way the conditional rendering was set up, some of the Boolean props in the local state of the `CurrentTrip` component were being toggled *after* the page re-routed back to the Form view.  So I just had to refactored the useEffect() chain a bit to update these local state props in the correct order.
- Resized the font in the "backup timer" in the AddTime modal on mobile views.

## How has this been tested?
- [x] In Browser
- [x] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

File: src/Components/CurrentTrip/CurrentTrip.js [hunk 1]
```diff
@@ -74,15 +74,20 @@ function CurrentTrip({ user, eta, contact, tripIsActive, setTripIsActive }) {
   useEffect(() => {
     if (emergency) {
       setAlertModalIsOpen(true);
-      setExtensionModalIsOpen(false);
       setEtaSeconds(null);
       setExtension(0);
-      // THIS IS CAUSING THE CONSOLE 'MEMORY LEAK' ERROR
```
Comment (carolineshowscode) [inline src/Components/CurrentTrip/CurrentTrip.js:80]: Nice job figuring this out

**Review by carolineshowscode — APPROVED** [review]
Looks good!

File: src/Components/CurrentTrip/CurrentTrip.js [hunk 2]
```diff
@@ -74,15 +74,20 @@ function CurrentTrip({ user, eta, contact, tripIsActive, setTripIsActive }) {
...
-      setTripEnded(true);
-      TripNotCompleteMessage(user, contact)
+      TripNotCompleteMessage(user, contact);
```
Comment (bhohnco) [inline src/Components/CurrentTrip/CurrentTrip.js:79]: great work here Pete!

File: src/Components/TripETA/TripETA.js [hunk 3]
```diff
@@ -97,8 +97,8 @@ function TripETA( { modalIsOpen, closeModal, eta, tripDetails, contact, userName
       {!loading && 
         <div className='eta-modal'>
           <p className='eta-message'>
-            <span>Your estimated trip time:</span>
-            <span>{etaString}</span>
+            <span className='eta-text'>Your estimated trip time:</span>
```
Comment (bhohnco) [inline src/Components/TripETA/TripETA.js:100]: Nice add for these. We'll be able to target them

**Review by bhohnco — APPROVED** [review]

**Pull request merged**